azext_iot/_help.py

```diff
@@ -912,3 +912,253 @@
     type: command
     short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
 """
+
+helps['iot dt'] = """
+    type: group
+    short-summary: Manage digital twin of an IoT Plug and Play device.
+"""
+
+helps['iot dt invoke-command'] = """
+    type: command
+    short-summary: Executes a command on an IoT Plug and Play device.
```

Comment:
This command and other `iot dt` based commands don't mentioned "Digital Twin" in their summaries. Do we want to update summaries (and example names) to include Digital Twin language?